docs/Chapter02-Overview-LoggingIn-and-Authentication.md

```diff
@@ -15,20 +13,18 @@ There’s a login for everything these days. The default way to prove to compute
 
 A quick step back. Security comes in many forms, not just digital. There are lots of aspects to consider when it comes to your personal security, and many things to focus in on when talking about security in the newsroom. A few things to keep in mind about security in general:
 
--   It’s unfortunate, but more often than not, good security takes a bit of effort. And in practice, there is often a trade-off of what is easy and what is most secure.
--   Good security does not rely on trust, though in practice it requires some.
+-   Good security takes a little bit of effort. There is sometimes a trade-off of what is easy and what is most secure.
 -   Though there are good general security practices, thinking about your security in terms of threat modeling (who will reasonably attack me, and what specifically do you need to defend against) will help focus your efforts.
--   Attacks are always changing, it’s good to stay up to date if just on a high level.
+-   Attacks are always changing, it’s good to stay up to date, even if just on a high level.
 -   Talk to experts. Unless you are totally confident of what you are doing, take a moment to double check with someone how to practice good security.
--   You are never 100% secure. Understand what you are risking.
+-   Nothing can be 100% secure, but we can be significantly more secure with minimal cost.
 -   Even if you don’t think you are at risk, we are all avenues to other people who may be targets. [*You are worthy of a data breach*](https://medium.com/@mshelton/journalists-you-are-worthy-of-a-data-breach-55f8e53fd3fe).
 
 ## Some terminology
 
 Just to make sure we are all on the same page, here are some top-level words and concepts:
 
 -   **Authentication**: A mechanism used to identify someone. The most common mechanism is a username and password, but there are others, such as PIN codes, fingerprints, and more.
--   **Authorization**: The ability to access a particular resource. For instance, your user on your computer has access to edit certain files, but not core, system files.
```

Comment:
It's not explicitly used in existing lesson plans but it could be in future ones, and it is a key concept. I'd vote for keeping it.

Reply:
No problem, adding that back in!